diff --git a/node_modules/vite-plugin-checker/dist/esm/checkers/typescript/main.js b/node_modules/vite-plugin-checker/dist/esm/checkers/typescript/main.js
index 8ed95d4..9548b24 100644
--- a/node_modules/vite-plugin-checker/dist/esm/checkers/typescript/main.js
+++ b/node_modules/vite-plugin-checker/dist/esm/checkers/typescript/main.js
@@ -2,7 +2,7 @@ import os from "node:os";
 import path from "node:path";
 import { fileURLToPath } from "node:url";
 import { parentPort } from "node:worker_threads";
-import chalk from "chalk";
+import picocolors from 'picocolors';
 import invariant from "tiny-invariant";
 import ts from "typescript";
 import { Checker } from "../../Checker.js";
@@ -79,9 +79,9 @@ const createDiagnostic = (pluginConfig) => {
             logChunk = "";
           }
           if (terminal) {
-            const color = errorCount && errorCount > 0 ? "red" : "green";
+            const color = "white";
             consoleLog(
-              chalk[color](
+              picocolors[color](
                 logChunk + os.EOL + wrapCheckerSummary(
                   "TypeScript",
                   diagnostic.messageText.toString()
